src/styles/service_order.css

```diff
@@ -79,6 +79,10 @@
     padding: 24px;
 }
 
+.service-list {
```

Comment:
```suggestion
.service-instance-list {
```
I suggest renaming the css name.

Reply:
fixed.